### ISSUE TITLE
multi-lang routing and 

### ISSUE THREAD

**Issue by proea** [skipperbent/simple-php-router#456]
Hi!

I'm trying to create multi language routing
And i face some issues

The example below will be slightly different from my real code, but this example accurately describes the problem and works identically to my problem.

My example:
```
Router::partialGroup(
    '/lang/{lang}/',
    function ($lang = 'en') {
        Router::match(['get', 'post'], '/', function () {
            return 'Hello world [get/post]';
        });
        Router::get('/test/', function () {
            return 'Hello world! (/test/) ' . Router::getUrl()->getAbsoluteUrl();
        });
        Router::group(['prefix' => '/auth'], function () {
            Router::get('/', function () {
                return 'Hello world! (/auth/test/) ' . Router::getUrl()->getAbsoluteUrl();
            });
            Router::get('/test/', function () {
                return 'Hello world! (/auth/test/) ' . Router::getUrl()->getAbsoluteUrl();
            });
        });
    }
);
```

Results:
```
/lang/en/ -> hello world
/lang/en/test/ -> hello world
/lang/en/auth/ -> NotFoundHttpException
/lang/en/auth/test/ -> NotFoundHttpException (Fatal error: Uncaught Pecee\SimpleRouter\Exceptions\NotFoundHttpException: Route not found: "/lang/en/auth/test/" in ..)
```

partialGroup( group() ) is not working for me

there is also no possibility to make alias for group or url (like symphony _Localized Routing (i18n)_)

Is there possibility to make multi-lang routing another way?
and implement the configuration above

**Comment by skipperbent** [comment]
Sorry for the late reply.

The issue with child-routes not loading correctly will be fixed in the next upcoming release.

**Comment by skipperbent** [comment]
This issue has been resolved in version 4.3.0.0.
Feel free to open another issue if you are still experiencing issues.
